### PR TITLE
Added atom threading extensions

### PR DESCRIPTION
**Pull request by icyerasor**
see http://purl.org/syndication/thread/1.0 - http://www.ietf.org/rfc/rfc4685.txt

### CODE REVIEW

**Comment by icyerasor** [comment]
Are there any reasons this does not get merged?

**Comment by PatrickGotthard** [comment]
Sorry, it's just because I have no time currently to maintain ROME. I just build a house and there are still plenty of little things to do.
Am 01.07.2015 7:10 nachm. schrieb icyerasor <notifications@github.com>:Are there any reasons this does not get merged?

**Comment by snoopdave** [comment]
Thanks for the contribution. I hope to review this over the next weekend.

File: src/main/java/com/rometools/modules/thr/ThreadingModule.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    public void setType(String type);
+
+    public String getHref();
+
+    public void setHref(String href);
+
+    public String getSource();
+
+    public void setSource(String href);
```
Comment (mishako) [inline src/main/java/com/rometools/modules/thr/ThreadingModule.java]: href -> source

**Comment by mishako** [comment]
:+1: apart from my comment

**Comment by snoopdave** [comment]
Thanks Icyerasor!

**Pull request merged**